### ISSUE TITLE
Odd indent size while editing python

### ISSUE THREAD

**Issue by onelivesleft** [slynch8/10x#1673]
While editing a python file, when I open a scope the next line indents by 2 spaces instead of 4.  For example: 
```python
def foo():
    if bar:
      |
```
The first indent correctly goes to 4 spaces, but the second  is only 2.

My current settings: 
```
# Width of a tab in chars
TabWidth:                              4

# Whether lines should be indented with tabs or spaces
# Valid values: Tabs, Spaces
IndentMode:                            Spaces

# Whether lines should be indented with tabs or spaces
# Valid values: Tabs, Spaces
PythonIndentMode:                      Spaces

# Automatically detect the indentation mode (tabs or spaces) when the files is opened. If inconclusive defaults to the IndentMode setting
AutoDetectIndentMode:                  false

# Auto-indent new lines
AutoIndent:                            true
```

**Comment by slynch8** [comment]
Is this still an issue? It's not happening for me anymore, and I don't remember fixing it

**Comment by slynch8** [comment]
fixed (1.0.35)
